pkg/apis/kubeone/helpers.go

```diff
@@ -52,6 +52,16 @@ func (c KubeOneCluster) Followers() []HostConfig {
 	return followers
 }
 
+func (c *KubeOneCluster) OwnNode(name string) bool {
```

Comment:
`OwnNode` is a little bit confusing name to me. I would propose that we rename it to something like `NodeExists`. Also, please add a comment describing what this function does.